.github/workflows/new_release.yml

```diff
@@ -0,0 +1,18 @@
+name: Run new-release script
+
+on: repository_dispatch
+
+jobs:
+  generate_diffs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: "12.x"
+      - name: Install Dependencies
+        run: yarn
```

Comment:
just a warning that this flakes surprisingly often, and should probably be cached, and should probably prefer offline or use lock file

```yaml

      - uses: actions/cache@v2
        name: Yarn Cache
        id: yarn-cache
        with:
          path: ${{ steps.workflow-variables.outputs.yarn-cache-dir }}
          key: ${{ runner.os }}-yarn-v1-${{ hashFiles('**/package.json') }}
          restore-keys: ${{ runner.os }}-yarn-v1


      - name: Yarn Install
        uses: nick-invision/retry@v2
        with:
          timeout_minutes: 10
          retry_wait_seconds: 60
          max_attempts: 3
          command: yarn --no-audit --prefer-offline
```

...is how I've been ...mostly reliably doing it in react-native-firebase

all the restore keys business lets you force expire the cache if needed (v1) but can restore well across only-slightly-different runs since it can use other caches that are close but not exact

the retry action is the best thing since sliced bread for github CI workflows that fetch external workflows. They can and will flake, this way they mostly flake and keep rolling vs stopping things dead

Reply:
(https://github.com/invertase/react-native-firebase/blob/main/.github/workflows/tests_e2e_ios.yml)